packages/javascript-sdk/src/fr-user/index.ts

```diff
@@ -41,27 +42,37 @@ abstract class FRUser {
    *
    * @param options Configuration overrides
    */
-  public static async logout(options?: ConfigOptions): Promise<void> {
+  public static async logout(options?: LogoutOptions): Promise<void> {
+    // Shallow copy options to delete redirect prop
+    const configOptions = { ...options };
+    delete configOptions.redirect;
+
     // Just log any exceptions that are thrown, but don't abandon the flow
     try {
       // Both invalidates the session on the server AND removes browser cookie
-      await SessionManager.logout(options);
+      await SessionManager.logout(configOptions);
     } catch (error) {
       FRLogger.warn('Session logout was not successful');
     }
+
+    try {
+      await OAuth2Client.revokeToken(configOptions);
```

Comment:
Is this specifically needed for the Ping redirect logout? I see we try/catch it so it won't break older flows, but i'm curious

Reply:
We just moved this up. There's no code change for this method call.